src/components/calcite-combobox/calcite-combobox.scss

```diff
@@ -1,31 +1,6 @@
 // scale
 // todo update when new spacing modifiers are introduced to calcite-base
 
-@mixin floatingContainer($state: "closed") {
```

Comment:
Did this not work for combobox / dropdown / other future floating container? I think the included css locations all changed in the last update, we could move it to `includes.scss` since it can be used across components?

Reply:
I'm not sure. I just saw it was removed in the dropdown PR so I just moved the styles from that code here.

Reply:
Ah, yeah it was only added in this PR. Can just add down the line 👍